server/src/main.rs

```diff
@@ -1,51 +1,30 @@
 use actix_web::{web, App, HttpResponse, HttpServer, Responder};
-use fightlines_moves::moves::Move;
-use std::collections::HashMap;
 use std::io;
 
-//
-struct Model {
-    games: HashMap<String, Game>,
-}
-
-struct Game {
-    name: String,
-    players: Vec<Player>,
-    round: i64,
-}
-struct Player {
-    name: String,
-    /// None means the player has not submitted moves for this turn
-    /// Some(...) means they have, including the possibility that
-    /// they submitted an empty list of moves.
-    moves: Option<Vec<Move>>,
-}
+use server::domain::model;
+use server::domain::model::Model;
 
-// Responder Objects
-// GET /
+/// Responder Objects
+/// GET /
 async fn index() -> impl Responder {
     HttpResponse::Ok().body("Hello World")
 }
 
-// GET /again
+/// GET /again
 async fn index2() -> impl Responder {
     HttpResponse::Ok().body("Hello World Again")
 }
 
-// GET /games/count, we also pass in the state
+/// GET /games/count, we also pass in the state
```

Comment:
I think triple forward slash is a Rust standard for documentation. It gets highlighted differently in my IDE too. Just FYI @VictorXLR

Reply:
oh, i didnt know that :), ill Change it moving forward